examples/examples_repositories.bzl

```diff
@@ -82,7 +82,7 @@ def include_examples_repositories():
         name = "nghttp2",
         build_file_content = all_content,
         patch_args = ["-p1"],
-        patch_cmds = ["find . -name '*.sh' -exec sed -i.orig '1s|#!/usr/bin/env sh\$|/bin/sh\$|' {} +"],
```

Comment:
Looks like here extra slashes were added, which is not correct...

Reply:
It is correct. Inside a string, `\$` is not correct. To insert a backslash, you need `\\$`.

Buildifier generated the changes automatically.

Reply:
I see, '\' is the escape symbol, for multiline strings also.
I am still worried that it will actually change the scripts.